lib/clif-backend/Cargo.toml

```diff
@@ -5,6 +5,9 @@ description = "Wasmer runtime Cranelift compiler backend"
 license = "MIT"
 authors = ["The Wasmer Engineering Team <engineering@wasmer.io>"]
 repository = "https://github.com/wasmerio/wasmer"
+keywords = ["wasm", "webassembly", "compiler", "JIT", "AOT"]
+categories = ["wasm"]
+maintainence = { status = "actively-developed" }
```

Comment:
Repeated typo, "maintenance" not "maintainence".

Reply:
I cannot spell that word 🙀